bl 2step -> zz_api

jeremyhermann opened the pull request.

gregseitz commented on app/controllers/albums_controller.rb:
```diff
@@ -1096,6 +1097,13 @@ def wizard
+    @album.cover = @album.photos.last
+    render :json => {:id => @album.cover.id, :t_url => @album.cover.thumb_url }, :status => 200, :layout => false and return
+  end
 
```
Just wanted to verify that this will be going away and replaced with what we discussed this morning with update album and current user info.

bowenli replied:
removed.

gregseitz commented on app/views/users/finish_profile.html.erb:
```diff
@@ -0,0 +1,416 @@
+        post_params: {'user_credentials': user_credentials, 'priority': 90},
+
+        // File Upload Settings
+        file_size_limit: '102400',    // 100MB
```
The server side max file upload is 40MB.

bowenli replied:
updated.

gregseitz commented on app/views/users/finish_profile.html.erb:
```diff
@@ -0,0 +1,455 @@
+                minlength : 1,
+                maxlength : 25,
+                regex : "(^[a-zA-Z0-9]+$|^[a-zA-Z0-9]+:.{8}$)",
+                remote : zz.routes.path_prefix + '/users/validate_username' // can add async true here to force validation on slow connection
```
We have a zz_api method that lets you validate username and/or email.  See /zz_api/users/available

jeremyhermann replied:
Let's get this release out and and not worry too much now about updating all the calls to use zz-api

Sent from my iPhone; sorry for spelling or terseness.

On Mar 13, 2012, at 6:50 PM, gregseitzreply@reply.github.com wrote:

> > +// takes a jquery element
> > +function add_profile_validation(element) {
> > -    zz.joinform.add_regex_validator();
> >   +
> > -    return element.validate( {
> > -        rules : {
> > -            'user[name]' : {
> > -                required : true,
> > -                minlength : 1
> > -            },
> > -            'user[username]' : {
> > -                required : true,
> > -                minlength : 1,
> > -                maxlength : 25,
> > -                regex : "(^[a-zA-Z0-9]+$|^[a-zA-Z0-9]+:.{8}$)",
> > -                remote : zz.routes.path_prefix + '/users/validate_username' // can add async true here to force validation on slow connection
> 
> We have a zz_api method that lets you validate username and/or email.  See /zz_api/users/available
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/zangzing/server/pull/108/files#r555475

Commit:
Fix done button progress indicator. Put back follow from join banner.

Commit:
Fix follow from join banner.
Raise exception to join screen.
Add more zza.
Fix cursor on header.
-------------------------------------
Merge branch 'jlh-2step' into bl-2step

Commit:
Merge branch 'jlh-2step' into bl-2step

gregseitz committed:
fix exception handler

Commit:
fix invite banner

Commit:
change finish profile top anchor position
